dynamic_rest/viewsets.py

```diff
@@ -418,36 +441,150 @@ def _bulk_update(self, data, partial=False):
         self.perform_update(serializer)
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    def _validate_patch_all(self, data):
+        if not isinstance(data, dict):
+            raise ValidationError(
+                'Patch-all data must be in object form'
+            )
+        serializer = self.get_serializer()
+        fields = serializer.get_all_fields()
+        validated = {}
+        for name, value in six.iteritems(data):
+            field = fields.get(name, None)
+            if field is None:
+                raise ValidationError(
+                    'Unknown field: "%s"' % name
+                )
+            source = field.source or name
+            if source == '*':
```

Comment:
Do we also need to check for `read_only` or would that happen upstream somewhere?

Reply:
Good catch, should check for it here